src/main/webapp/assets/modules/common/feedback/feedbackDirective.js

```diff
@@ -7,29 +7,31 @@ techlooper.directive("feedbackForm", function (apiService, $timeout) {
       composeEmail: "="
     },
     link: function (scope, element, attr, ctrl, composeEmail) {
-      if(scope.composeEmail.registrantLastName){
+      if (scope.composeEmail.registrantLastName) {
         scope.composeEmail.names = scope.composeEmail.registrantFirstName + ' ' + scope.composeEmail.registrantLastName;
-      }else{
+      }
+      else {
         scope.composeEmail.names = scope.composeEmail.registrantFirstName;
       }
 
-      scope.send = function(){
-        if(scope.feedbackContent == undefined || scope.feedbackContent == ''){
+      scope.send = function () {
+        if (scope.feedbackContent == undefined || scope.feedbackContent == '') {
```

Comment:
Use '===' to compare with 'undefined'.<br>Mixed double and single quotes.<br>Use '===' to compare with ''.